### ISSUE TITLE
SFML registry enables completion, github doesn't

### ISSUE THREAD

**Issue by brandonson** [phildawes/racer#507]
Completing for types/functions etc from sfml fails with this:
```
  [package]
  name = "tribulum"
  version = "0.1.0"
  
  [dependencies]
  artifact = "*" 
  rand = "*" 

  [dependencies.sfml]
  git = "ssh://git@github.com/jeremyletang/rust-sfml.git"
```

However, it works with this:
```
  [package]
  name = "tribulum"
  version = "0.1.0"
  
  [dependencies]
  artifact = "*" 
  rand = "*" 
  sfml = "*"
```

Oddly enough, I've tried other github/bitbucket repositories, and they appear to complete correctly.  The only thing I can think of is that sfml has a different folder name than the actual crate name (folder is `rust-sfml` while crate is just `sfml`). HOWEVER, I've tried with a different repository where the repo name is `<cratename>-rs` and it worked just fine.

In short, it really looks like it's attached to sfml specifically, but only when using the repo as a dependency rather than the cargo-registry version.

**Comment by brandonson** [comment]
Note: It does appear that the github and cargo versions of sfml don't quite match up in terms of API etc.  It's possible that something in the newer github versions just doesn't work with racer.
